project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss

```diff
@@ -20,3 +20,22 @@
 #headerHero {
   background: none;
 }
+
+#block-nicsloginnotice {
+  background: $grey-medium;
+  text-align: center;
+
+  h2 {
+    @include adjust-font-size-to(30px);
+  }
+
+  p {
+    @include adjust-font-size-to(16px);
+  }
+}
+#header {
+  background: #f8f8f8;
+  border: solid 1px #e7e7e7;
+  border-radius: 6px;
+  padding: 5px 5px 0px 20px;
```

Comment:
If a value is 0 it doesn't need px so should be `padding: 5px 5px 0 20px;`
Might actually be better as rem too `padding: .5rem .5rem 0 2rem;`?

Reply:
Done thanks Adam